tailscale.yaml

```
@@ -34,6 +34,16 @@ pipeline:
 
   - uses: strip
 
+subpackages:
+  - name: ${{package.name}}-compat
+    description: "Compatability package to provide parity with upstream Tailscale binary locations"
+    pipeline:
+      - runs: |
+          mkdir -p ${{targets.subpkgdir}}/usr/local/bin
+          ln -sf ${{targets.destdir}}/usr/bin/containerboot ${{targets.subpkgdir}}/usr/local/bin/containerboot
+          ln -sf ${{targets.destdir}}/usr/bin/tailscale ${{targets.subpkgdir}}/usr/local/bin/tailscale
+          ln -sf ${{targets.destdir}}/usr/bin/tailscaled ${{targets.subpkgdir}}/usr/local/bin/tailscaled
```

```suggestion
          ln -sf ${{targets.subpkgdir}}/usr/bin/containerboot ${{targets.subpkgdir}}/usr/local/bin/containerboot
          ln -sf ${{targets.subpkgdir}}/usr/bin/tailscale ${{targets.subpkgdir}}/usr/local/bin/tailscale
          ln -sf ${{targets.subpkgdir}}/usr/bin/tailscaled ${{targets.subpkgdir}}/usr/local/bin/tailscaled
```
If we use destdir then it'll point to a different location, we should be using subpkgdir in here for symlink to work correctly. 
You can see [here](https://apk.chaindag.dev/https/apk.cgr.dev/wolfi-presubmit/6387adc60c998862607a026128fef6ec7479b6df/x86_64/tailscale-compat-1.80.0-r1.apk@etag:ed9afa8c370b371af01630e6255271f4/usr/local/bin/) that it's pointing to some file in `/home/melange-out` which will not be there during install time so symlink will error out.

---

Implemented a variant of this in [`f604c4d` (#41327)](https://github.com/wolfi-dev/os/pull/41327/commits/f604c4de5ac8ad5f74972801ff2cef7a13c8d998).